### PR TITLE
refactor(*.js): rename shouldComponentRenderCSS/HTML to shouldRenderCSS/HTML

### PR DESCRIPTION
**Pull request by ASchmidt1024**
- search and replace
- smoke test of all components

---

refactor(arrow.js, breadcrumb.js): rename shouldComponentRenderCSS to shouldRenderCSS and shouldComponentRenderHTML to shouldRenderHTML

refactor(Button.js, EMail.js): rename shouldComponentRenderCSS to shouldRenderCSS and shouldComponentRenderHTML to shouldRenderHTML for consistency

refactor(EmotionPictures.js, GoogleMaps.js): rename shouldComponentRenderCSS to shouldRenderCSS and shouldComponentRenderHTML to shouldRenderHTML

refactor: rename shouldComponentRenderCSS to shouldRenderCSS in Hotspot.js and IconAmp.js
refactor: rename shouldComponentRenderHTML to shouldRenderHTML in Hotspot.js and IconAmp.js

refactor(IconLocation.js, IconPaperclip.js): rename shouldComponentRenderCSS to shouldRenderCSS and shouldComponentRenderHTML to shouldRenderHTML

refactor(Iframe.js, Input.js): rename shouldComponentRenderCSS to shouldRenderCSS and shouldComponentRenderHTML to shouldRenderHTML for consistency and clarity

refactor: rename shouldComponentRenderCSS to shouldRenderCSS in Link.js and Loading.js
refactor: rename shouldComponentRenderHTML to shouldRenderHTML in Link.js and Loading.js

refactor(Logo.js, MenuIcon.js): rename shouldComponentRenderCSS to shouldRenderCSS and shouldComponentRenderHTML to shouldRenderHTML for consistency and clarity

refactor(Picture.js, SliderButton.js): rename shouldComponentRenderCSS to shouldRenderCSS and shouldComponentRenderHTML to shouldRenderHTML for consistency and clarity

refactor(TotalPrice.js, Video.js): rename shouldComponentRenderCSS to shouldRenderCSS and shouldComponentRenderHTML to shouldRenderHTML for consistency and clarity

refactor(News.js, NewsList.js): rename shouldComponentRenderCSS to shouldRenderCSS for consistency

refactor(NewsPreview.js, TagManager.js): rename shouldComponentRenderHTML to shouldRenderHTML and shouldComponentRenderCSS to shouldRenderCSS

refactor(Widget.js, Carousel.js): rename shouldComponentRender to shouldRender for consistency and clarity

refactor(carouselTwo.js, cookieBanner.js): rename shouldComponentRenderCSS to shouldRenderCSS and shouldComponentRenderHTML to shouldRenderHTML for consistency and clarity

refactor(Details.js, Flockler.js): rename shouldComponentRenderCSS to shouldRenderCSS and shouldComponentRenderHTML to shouldRenderHTML for consistency and clarity

refactor(Form.js, ImageHotspot.js): rename shouldComponentRenderCSS to shouldRenderCSS and shouldComponentRenderHTML to shouldRenderHTML for consistency

refactor(Navigation.js, Pagination.js): rename shouldComponentRenderCSS to shouldRenderCSS and shouldComponentRenderHTML to shouldRenderHTML for consistency and clarity

refactor(PictureText.js, PictureWithPicture.js): rename shouldComponentRenderCSS to shouldRenderCSS and shouldComponentRenderHTML to shouldRenderHTML for consistency

refactor(RecipeList.js, TagFilter.js): rename shouldComponentRenderCSS to shouldRenderCSS and shouldComponentRenderHTML to shouldRenderHTML

refactor(Teaser.js): rename shouldComponentRenderCSS to shouldRenderCSS and update its usage in the code

refactor(CookieBanner.js, Login.js): rename shouldComponentRenderHTML to shouldRender for consistency and clarity

refactor(ProductList.js, QuestionsAnswers.js): rename shouldComponentRender to shouldRender

refactor(RatingsReviews.js, ReviewsSummary.js, StoreFinder.js): rename shouldComponentRender method to shouldRender

refactor(Body.js): rename shouldComponentRenderCSS to shouldRenderCSS and shouldComponentRenderHTML to shouldRenderHTML.

refactor(BodyStyle.js): rename shouldComponentRenderCSS to shouldRenderCSS and intersectionShouldComponentRenderCSS to intersectionshouldRenderCSS for consistency and readability

refactor(organisms): rename shouldComponentRenderCSS to shouldRenderCSS and shouldComponentRenderHTML to shouldRenderHTML in Footer.js and Grid.js components

refactor(Header.js): rename shouldComponentRenderCSS to shouldRenderCSS and shouldComponentRenderHTML to shouldRenderHTML, to improve code readability and consistency

refactor(modal.js, wrapper.js): rename shouldComponentRenderCSS to shouldRenderCSS and shouldComponentRenderHTML to shouldRenderHTML for consistency

refactor(General.js): rename shouldComponentRenderCSS to shouldRenderCSS and update its usage in connectedCallback and shouldRenderCSS method.

### CODE REVIEW

File: src/es/components/organisms/bodyStyle/BodyStyle.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ export default class BodyStyle extends Intersection(Body) {
   intersectionCallback (entries, observer) {
     // render css on intersection because this component is often used for backgrounds including css backgrounds with images. those can by default not be loaded lazy nor support sources, thats why we load them on intersection
     if ((this.isIntersecting = entries && entries[0] && entries[0].isIntersecting)) {
-      if (this.intersectionShouldComponentRenderCSS()) {
+      if (this.intersectionshouldRenderCSS()) {
```
Comment (Weedshaker) [inline src/es/components/organisms/bodyStyle/BodyStyle.js:33]: intersectionshouldRenderCSS => intersection**S**houldRenderCSS

File: src/es/components/organisms/bodyStyle/BodyStyle.js [hunk 2]
```diff
@@ -56,7 +56,7 @@ export default class BodyStyle extends Intersection(Body) {
    *
    * @return {boolean}
    */
-  intersectionShouldComponentRenderCSS () {
+  intersectionshouldRenderCSS () {
```
Comment (Weedshaker) [inline src/es/components/organisms/bodyStyle/BodyStyle.js:59]: intersectionshouldRenderCSS => intersection**S**houldRenderCSS

**Review by Weedshaker — CHANGES_REQUESTED** [review]
camel case issue at BodyStyle

**Comment by Weedshaker** [comment]
oh, and you could check the box or actually remove the todo bullet point! THX

File: src/es/components/organisms/bodyStyle/BodyStyle.js (hunk 1 above)
Comment (ASchmidt1024) [reply src/es/components/organisms/bodyStyle/BodyStyle.js]: Good catch.

**Pull request merged**